tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -81,9 +131,9 @@ else
     sed -i "s|^webdav.ui.enabled = .*|webdav.ui.enabled = False|g" /etc/tracim/development.ini
     sed -i "s|^\s*Define START_WEBDAV|    # Define START_WEBDAV|g" /etc/tracim/apache2.conf
 fi
-
 # Activate or deactivate caldav
 if [ "$START_CALDAV" = "1" ]; then
+    log "Activation of caldav"
```

Comment:
Maybe log can explain `Creation of caldav configuration file` or something like that.